package.json

```diff
@@ -5,14 +5,11 @@
   "bin": {
     "prettier": "./bin/prettier.js"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/jlongster/prettier.git"
-  },
+  "repository": "prettier/prettier",
   "author": "James Long",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/jlongster/prettier/issues"
+    "url": "https://github.com/prettier/prettier/issues"
```

Comment:
Since you changed "repository" to using the github shortcut, the entire "bugs" field can be removed – `npm bugs` works it out automatically.